table/plain/plain_table_builder.h

```diff
@@ -45,7 +45,8 @@ class PlainTableBuilder: public TableBuilder {
       const std::string& column_family_name, uint32_t num_probes = 6,
       size_t huge_page_tlb_size = 0, double hash_table_ratio = 0,
       bool store_index_in_file = false, const std::string& db_id = "",
-      const std::string& db_session_id = "", uint64_t file_number = 0);
+      const std::string& db_session_id = "", uint64_t file_number = 0,
+      int level_at_creation = -1);
```

Comment:
Maybe use the same constant that you plan to define for unknown level in table_properties.h

Reply:
Discussed offline: it turns out that, since `PlainTableBuilder` is a orthogonal concept to `TablePropertiesCollectorFactory::Context` it might not be ideal to adopt this suggestion (i.e,`int level_at_creation = TablePropertiesCollectorFactory::Context::kUnknownLevelAtCreation`). However, @anand1976 has suggested to make `level_at_creation` a required parameter for `PlainTableBuilder`'s constructor to (1) avoid the need for such a constant from the first place and (2) makes sure the correct `level_at_creation` is always passed into `PlainTableBuilder`'s constructor. 

This suggestion is adopted and will be reflected in the PR update.